src/fsharp/FSharp.Compiler.nuget/Microsoft.FSharp.Compiler.nuspec

```diff
@@ -40,13 +39,16 @@
         </dependencies>
     </metadata>
     <files>
-        <file src="fsc.exe"                           target="lib/netstandard1.6" />
-        <file src="FSharp.Compiler.dll"               target="lib/netstandard1.6" />
-        <file src="default.win32manifest"             target="runtimes/any/native/" />
-        <file src="FSharp.Build.dll"                  target="lib/netstandard1.6" />
-        <file src="Microsoft.FSharp.targets"          target="runtimes/any/native/" />
-        <file src="Microsoft.Portable.FSharp.targets" target="runtimes/any/native/" />
-        <file src="Microsoft.FSharp.NetSdk.targets"   target="runtimes/any/native/" />
-        <file src="Microsoft.FSharp.NetSdk.props"     target="runtimes/any/native/" />
+        <file src="coreclr/bin/fsc.exe"                           target="lib/netstandard1.6" />
+        <file src="coreclr/bin/fsi.exe"                           target="lib/netstandard1.6" />
+        <file src="coreclr/bin/fsharp.core.dll"                   target="lib/netstandard1.6" />
```

Comment:
"FSharp.Core" capitalization please - the lowercase naming should never appear for the DLL  (otherwise I assume this will bite us on Linux?)